packages/cruntime/Dockerfile

```diff
@@ -0,0 +1,81 @@
+# (c) Cartesi and individual authors (see AUTHORS)
+# SPDX-License-Identifier: Apache-2.0 (see LICENSE)
+
+# syntax=docker.io/docker/dockerfile:1
+ARG IMAGE_REGISTRY
+ARG IMAGE_NAMESPACE
+ARG IMAGE_NAME
+ARG IMAGE_TAG
+
+###############################################################################
+# STAGE: base-image
+#
+# This stage creates a base-image with apt repository cache and ca-certificates
+# to be used by later stages.
+#
+FROM ${IMAGE_REGISTRY}/${IMAGE_NAMESPACE}/${IMAGE_NAME}:$IMAGE_TAG AS base-image
+ARG DEBIAN_FRONTEND=noninteractive
+RUN apt-get update && apt-get install -y --no-install-recommends ca-certificates
+
+###############################################################################
+# STAGE: chisel
+#
+# Build the chiselled filesystem based on the desired slices.
+# This image should have the machine-emulator-tools and crun dependencies
+# installed.
+#
+FROM base-image AS chisel
+
+WORKDIR /rootfs
+
+# Extract machine-emulator-tools into the chiselled filesystem
+ARG MACHINE_EMULATOR_TOOLS_VERSION
+ADD https://github.com/cartesi/machine-emulator-tools/releases/download/v${MACHINE_EMULATOR_TOOLS_VERSION}/machine-emulator-tools-v${MACHINE_EMULATOR_TOOLS_VERSION}.deb /
+RUN dpkg -x /machine-emulator-tools-v${MACHINE_EMULATOR_TOOLS_VERSION}.deb /rootfs
+
+# Get chisel binary
+ARG CHISEL_VERSION
+ADD "https://github.com/canonical/chisel/releases/download/v${CHISEL_VERSION}/chisel_v${CHISEL_VERSION}_linux_riscv64.tar.gz" chisel.tar.gz
+RUN tar -xvf chisel.tar.gz -C /usr/bin/
+
+# Extract crun dependencies into the chiselled filesystem
+# FIXME: remove this when crun's dependecies slices are upstream
+ADD https://github.com/endersonmaia/chisel-releases.git#14cd0571ac7646c0e2b70e3dc41855b70805d7eb /chisel-24.04
+RUN chisel cut \
+    --release /chisel-24.04 \
+    --root /rootfs \
+    --arch=riscv64 \
+    # base rootfs dependencies
+    base-files_base \
+    base-files_release-info \
+    base-passwd_data \
+    # machine-emulator-tools dependencies
+    busybox-static_bins \
+    libgcc-s1_libs \
```

Comment:
is this explicitly required, or should come by transitive dependency?

Reply:
The installation of mahine-emulator-tools is made outside the chisel process, so we need to do this way.

Reply:
But doesn't the debian installation of the tools bring what is needed?

Reply:
I'm not installing, but just extracting its contents.

If I install it, it's gonna pollute the `/rootfs` that should be minimal.